imported/spirv/spvgen.h

```diff
@@ -28,6 +28,7 @@
  * @brief SPVGEN header file: contains the definition and the wrap implementation of SPIR-V generator entry-points.
  ***********************************************************************************************************************
  */
+// clang-format off
```

Comment:
I think this should be turned on at the end of the file. I'm not sure if this will disable formatting in all files that include this header or not, but because in C/C++ includes are textual, this looks suspicious.

Reply:
good point. will do it